cpp/ps0001/problem 1.cpp

```diff
@@ -1,20 +1,23 @@
 #include <iostream>
 using namespace std;
 int main(){
-    int num1;
-    int num2;
-    int num3;
-    int sum;
+    // prompting user for 3 numbers to add 1/2
+    short num1;
+    short num2;
+    short num3;
+    short sum;
     cout <<"please enter first number\n";
     cin >>num1;
     cout <<"please enter second number\n";
     cin >>num2;
     cout <<"please enter third number\n";
     cin >>num3;

```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add